token/program-2022/src/lib.rs

```diff
@@ -35,6 +35,55 @@ pub fn amount_to_ui_amount(amount: u64, decimals: u8) -> f64 {
     amount as f64 / 10_usize.pow(decimals as u32) as f64
 }
 
+/// Convert a raw amount to its UI representation (using the decimals field defined in its mint)
```

Comment:
Just to be explicit, these functions are duplicated so that we can move forward with the release/deploy before releasing a new spl_token?

Reply:
Yes exactly. I know it's not ideal, but releasing the token crate is such a hassle that duplicating was by far easier than deploying and all that.